src/network-analytics/azext_network_analytics/tests/user_identity.json

```diff
@@ -0,0 +1,6 @@
+{"/subscriptions/9a021c4c-ddcd-4e80-9155-a5b0e872f4c2/resourceGroups/a40rama/providers/Microsoft.ManagedIdentity/userAssignedIdentities/ramatestiden":
+    {
+    "principalId": "00000000-0000-0000-0000-000000000000",
+    "clientId": "00000000-0000-0000-0000-000000000000"
+    }
+}
```

Comment:
Please move this file under `latest` folder.

Reply:
Removed unused file.